internal/config/options.go

```diff
@@ -254,6 +283,23 @@ func (o *Options) Validate() error {
 		o.Headers = make(map[string]string)
 	}
 
+	if o.InsecureServer {
+		log.Warn().Msg("InsecureServer mode: WE _STRONGLY_ ADVISE AGAINST USING " +
+			"POMERIUM WITHOUT ENCRYPTION! In this configuration, it is up to you " +
+			"to secure your instance and handle TLS termination. If exposing HTTP " +
+			"directly, you must also set `cookie_secure` to false")
+	} else if o.Cert != "" && o.Key != "" {
+		o.TLSCertificate, err = cryptutil.CertifcateFromBase64(o.Cert, o.Key)
+	} else if o.CertFile != "" && o.KeyFile != "" {
+		o.TLSCertificate, err = cryptutil.CertificateFromFile(o.CertFile, o.KeyFile)
+	}
+	// else {
+	// 	err = errors.New("no certificates supplied nor was insecure mode set")
```

Comment:
Should we fail hard if neither certificates nor use insecure is set? Part of me thinks yes. I was considering falling back on insecure but that feels wrong.

Reply:
I think that's the right call.  Explicit security relaxation is pretty common (docker and curl come to mind).